src/containers/layout/AppHeader.js

```diff
@@ -7,29 +7,31 @@ import { routes } from '~/constants/routes'
 const AppHeader = () => {
   const { t } = useTranslation()
   
-  return (<AppBar position="static">
-    <Toolbar>
-      <IconButton
-        aria-label="menu"
-        color="inherit"
-        edge="start"
-        size="large"
-        sx={ { mr: 2 } }
-      >
-        <MenuIcon />
-      </IconButton>
-      <Button
-        component={ Link } to={ routes.home }
-        variant="h6"
-      >
-        { t('common.title') }
-      </Button>
-      <Box flexGrow={ 1 } />
-      <Button color="inherit" component={ Link } to={ routes.about }>
-        { t('common.about') }
-      </Button>
-    </Toolbar>
-  </AppBar>)
+  return (
+    <AppBar position="static">
```

Comment:
You should create a general nav component that will be built depending on incoming props. And then create guest configuration for it.